MapleServer2/Types/Trophy.cs

```diff
@@ -65,7 +64,7 @@ public Trophy(long uid, int trophyId, int nextGrade, long counter, bool isDone,
         AccountId = accountId;
     }
 
-    public void AddCounter(GameSession session, long amount)
+    public void AddCounter(Player player, long amount)
```

Comment:
This is a great example of passing around the minimum required class to perform an operation!

I see this a lot in server emulators in general - because people _can_ pass around an entire Session (or equivalent) object they will, when in reality the consumer only needs a small part of that object.

We have a few examples around the server where we only need an inventory, but we pass in an entire player object, and the first thing we do is get the inventory from the player!

Great job seeing this and improving it :)